kaiko/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'EXAMPLE'
```

Comment:
```suggestion
const NAME = 'KAIKO'
```

@krebernisak Do we want this capitalized or lowercase? I've seen both ways, but let's pick a convention.

Reply:
This was first used as an ENV prefix to avoid naming collisions when using two or more adapters in the same env. So for now it was a simple hardcoded uppercase string

What we should do now to make this more robust is this:

- Import the `name` from `package.json`, remove the `-adapter` suffix, and export that as NAME.
- Create a `toEnvName` util function that will map it to a safe string to use as ENV name

> Environment variable names used by the utilities in the Shell and Utilities volume of IEEE Std 1003.1-2001 consist solely of uppercase letters, digits, and the '_' (underscore) from the characters defined in Portable Character Set and do not begin with a digit.